webserver/src/stm_comms/stm_comms.cpp

```diff
@@ -172,3 +172,13 @@ void stmCommsSendUpdateOperationMode(const OperationMode mode) {
   );
   xSemaphoreGiveRecursive(mcucLock);
 }
+
+void stmCommsSendUpdateSystemState(const SystemState& state) {
+  if (xSemaphoreTakeRecursive(mcucLock, portMAX_DELAY) == pdFALSE) return;
+  SystemState command = { .tarePending = state.tarePending };
```

Comment:
Given its name I would expect it to also send the operation mode?

Reply:
Wasn't in global thinking mode soz, had a single task in mind lol